opensfm/align_pdr.py

```diff
@@ -274,6 +274,13 @@ def direct_align_pdr(data, target_images=None):
     :param target_images:
     :return: reconstruction
     """
+    if data.pdr_shots_exist():
```

Comment:
is there a danger of this failing if gps picker selects 'hybrid'? I'm thinking we should impose a limit in the gps picker to avoid a problem in processing

Reply:
RI-625 is the ticket for the gps picker tool to handle the no movement case. I will edit that ticket, so it emphasizes that, when there's no movement (pdr_shots is empty), then the gps picker tool must set the alignment_mode to "direct".